solutions/quickstart/variables.tf

```diff
@@ -0,0 +1,251 @@
+      alltrue([
+        can(regex("^[a-z][-a-z0-9]*[a-z0-9]$", var.prefix)),
+        length(regexall("--", var.prefix)) == 0
+      ])
+    )
+    error_message = "Prefix must begin with a lowercase letter and may contain only lowercase letters, digits, and hyphens '-'. It must not end with a hyphen('-'), and cannot contain consecutive hyphens ('--')."
+  }
+
+  validation {
+    # must not exceed 16 characters in length
+    condition     = length(var.prefix) <= 16
+    error_message = "Prefix must not exceed 16 characters."
+  }
+}
+
+variable "existing_resource_group_name" {
+  type        = string
+  description = "The name of an existing resource group to provision the cluster."
+  default     = "Default"
+}
+
+variable "region" {
+  type        = string
+  description = "The region to provision all resources in. [Learn more](https://terraform-ibm-modules.github.io/documentation/#/region) about how to select different regions for different services."
+  default     = "us-south"
+}
+
+variable "vpc_resource_tags" {
+  type        = list(string)
+  description = "Metadata labels describing this vpc deployment, i.e. test."
+  default     = []
+}
+
+variable "cluster_resource_tags" {
+  type        = list(string)
+  description = "Metadata labels describing this cluster deployment, i.e. test."
+  default     = []
+}
+
+variable "access_tags" {
+  type        = list(string)
+  description = "A list of access tags to apply to the resources created by the module."
+  default     = []
+}
+
+##############################################################################
+# VPC variables
+##############################################################################
+
+variable "address_prefix" {
+  description = "The IP range that will be defined for the VPC for a certain location. Use only with manual address prefixes."
+  type        = string
+  default     = "10.10.10.0/24"
+}
+
+variable "zone" {
+  type        = number
+  description = "Specify the zone to which the cluster will be deployed."
+  default     = 1
+  validation {
+    condition     = contains([1, 2, 3], var.zone)
+    error_message = "Each region has only 3 zones."
+  }
+}
+##############################################################################
+# Cluster variables
+##############################################################################
+
+variable "cluster_name" {
+  type        = string
+  description = "The name of the new IBM Cloud OpenShift Cluster. If a `prefix` input variable is specified, it is added to this name in the `<prefix>-value` format."
+  default     = "openshift"
+}
+
+variable "ocp_version" {
+  type        = string
+  description = "Version of the OCP cluster to provision."
+  default     = "4.17"
+
+  validation {
+    condition     = tonumber(var.ocp_version) >= 4.17
+    error_message = "To install Red Hat OpenShift Virtualization, all `ocp_version` should be higher than `4.17`."
```

Comment:
I think it should be - 
`all `ocp_version` should be equal to or higher than `4.17``